Wormholy.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Wormholy"
-  s.version      = "1.5.1"
+  s.version      = "1.5.7"
```

Comment:
Why you bumped the version to 1.5.7 from 1.5.1? However, I suggest you remove this part, then I will modify the version when it is ready to go to Cocoapods.

Reply:
I'm sorry :( When I started the fork, I didn't think to ask a pull request... I revert the change